changes/issue2988.yaml

```diff
@@ -0,0 +1,2 @@
+enhancement:
+  - "Make use of `kubernetes` extra logger in the `DaskKubernetesEnvironment` optional - [#2988](https://github.com/PrefectHQ/prefect/issues/2988)"
```

Comment:
On the fence about whether to include this as both an enhancement _and_ a breaking change in case anyone relied on these logs?

Reply:
Yeah it could go there since it is a slight change in that regard, will add it